addons/sale_timesheet/controllers/main.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.http import request
+from odoo.tools import float_round
+
+from odoo.addons.hr_timesheet.controllers.main import TimesheetController
+
+
+class SaleTimesheetController(TimesheetController):
+
+    def _prepare_plan_values(self, domain):
+        values = super(SaleTimesheetController, self)._prepare_plan_values(domain)
+        billable_types = ['non_billable', 'billable_time', 'billable_fixed']
+
+        # -- Dashboard (per billable type)
+        dashboard_values = {
+            'hours': dict.fromkeys(billable_types + ['total'], 0.0),
+            'rates': dict.fromkeys(billable_types + ['total'], 0.0),
+            'profitability': {
+                'invoiced': 0.0,
+                'to_invoiced': 0.0,
+                'cost': 0.0,
+                'total': 0.0,
+            }
+        }
+        dashboard_domain = domain + [('timesheet_invoice_type', '!=', False)]  # force billable type
+        dashboard_data = request.env['account.analytic.line'].read_group(dashboard_domain, ['unit_amount', 'timesheet_revenue', 'timesheet_invoice_type'], ['timesheet_invoice_type'])
+
+        dashboard_total_hours = sum([data['unit_amount'] for data in dashboard_data])
+        for data in dashboard_data:
+            billable_type = data.get('timesheet_invoice_type')
+            # hours
+            dashboard_values['hours'][billable_type] = float_round(data.get('unit_amount'), precision_rounding=0.01)
```

Comment:
It's simply to have only 2 number for the decimal part. With the float_time widget, you can have 4.3333333333333333 hours stored in database, and when display on Plan, it's not beautiful and I can not add padding.
It's just a question of display, since it is more important than having good number ^^